config/src/config.rs

```diff
@@ -303,6 +360,13 @@ pub struct Storage {
     pub master_key_import_path: Option<PathBuf>,
 }
 
+fn to_protected_string<S>(_val: &Option<Protected>, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: serde::Serializer,
+{
+    serializer.serialize_str("**** REDACTED ****")
```

Comment:
This should check if the password is `None`, because in that case we must keep it as `None`

Reply:
I modified this part as you suggested in the latest commit 2a6fc17. Could you review new code? Thanks in advance.